final-project/src/tier1/main.py

```diff
@@ -47,55 +45,68 @@ def __init__(self, debug=False):
         # Moving setpoint time counter
         self.moving_setpoint_time_counter = 0.0
         self.moving_setpoint_time_counter_increment = 0.5
+        self.env.reset_idx([0])
 
     def choose_new_goal_position(self):
         """Chooses a new goal position"""
         # Choose a random index from the goal position options
         # self.goal_position = self.goal_position_options[
         #     torch.randint(0, self.goal_position_options.shape[0], (1,)).item()]
         # Just choose a coordinate along the x-axis
-        self.goal_position = torch.tensor([19.0, 0.0, 7.5],
+        self.goal_position = torch.tensor([19.0, 19.0, .2],
```

Comment:
Probably needs to be reverted to a point along the X axis. Z coord can be whatever

Reply:
To add on, the reasoning behind along the x axis is because of the orientation of the maps. Along the x axis means the goal is at the end of the maze

Reply:
I had tested this and it learned to do the diagonal, I also understand the setpoint is not being set on diagonal and will revert to x-axis as that is also under testing.

Reply:
Created https://github.com/WPI-RL/drl_quadcopter/issues/14 for it.